app/views/admin/help/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<% content_for :page_title, 'Help' %>
+
+<div class="grid-row">
+  <div class="column-full">
+    <h1 class="heading-xlarge">
+      Help
+    </h1>
+
+    <div id="content">
+      <%= render partial: "admin/help/#{@case_management.to_partial_path}" %>
```

Comment:
I wanted this to just be `<%= render @case_management %>` but didn't want to add `admin/help` at the model layer, and can't figure out how to get the correct lookup context without writing it as above.